modules/ghlint-api/src/main/kotlin/net/twisterrob/ghlint/model/Permission.kt

```diff
@@ -0,0 +1,49 @@
+package net.twisterrob.ghlint.model
+
+public enum class Permission(public val value: String) {
+	ACTIONS("actions"),
+	ATTESTATIONS("attestations"),
+	CHECKS("checks"),
+	CONTENTS("contents"),
+	DEPLOYMENTS("deployments"),
+	ID_TOKEN("id-token"),
+	ISSUES("issues"),
+	METADATA("metadata"),
+	PACKAGES("packages"),
+	PAGES("pages"),
+	PULL_REQUESTS("pull-requests"),
+	REPOSITORY_PROJECTS("repository-projects"),
+	SECURITY_EVENTS("security-events"),
+	STATUSES("statuses")
+}
+
+public enum class Access(public val value: String) : Comparable<Access> {
+	// Order determines restrictiveness of permission
```

Comment:
```suggestion
	// Order determines restrictiveness of permission: none < read < write.
```
btw, this might be yagni, is it still used? (Although it's quite cool :)

(btw, Enum is comparable by default, but nice to "document" it explicitly by explicit interface)